.gitattributes

```
@@ -1,2 +1,2 @@
-.java text=auto
-.xml text=auto
+*.java text=auto
+*.xml text=auto
```

I think we should change this in a separate PR.
```suggestion
.java text=auto
.xml text=auto
```

---

With some struggle... reverted the .gitattribute file and created a separate issue: #10472